docs/cloud/cloud-databases/cloud-db-serverless-scale.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Scale a serverless database
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 3
+---
+
+# How do I scale a serverless up or down database?
+{: .no_toc }
+
+{% include /serverless/serverless-preview-warning.md %}
+
+This procedure explains how to scale a serverless database up or down. 
```

Comment:
Once again, scale implies direction.

I created an explanation in the sample /cloud-databases/cloud-db-serverless-home.md which would probably work here. It's under the heading "How do I scale database compute resources without down-time"